XmlParser/src/ua/goit/alg/parser/StateMachine.java

```diff
@@ -0,0 +1,270 @@
+package ua.goit.alg.parser;
+
+public class StateMachine {
```

Comment:
I guess you don't want it public. The only two things to be visible outside of the package which I understand from the test examples are xmlparser & handler interface